### DISCUSSION TITLE
Hi from Nicaragua

### DISCUSSION THREAD

**Discussion by Herald-Flores** [gittogethers/octogatosconf22#54] [Where in the world?]
Hi everyone, I'm Herald Flores 👋

I am very excited to learn new things every day, very happy for these spaces to continue learning about this beautiful community #OctoGatosConf 🥳.

**Comment by AndreaGriffiths11** [comment]
Bienvenido!

**Comment by Dreamer-byte** [comment]
Hola, ¿Cómo estan?.
Yo con muchas expectativas del evento.

**Reply by AndreaGriffiths11** [reply]
Ya casito!!! ⭐️⭐️⭐️ gracias por unirte
